<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoreInfo</class>
 <widget class="QDialog" name="MoreInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>731</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEdit_facts">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="button_nextFact">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Next Fact</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_addFact">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Add Fact</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_deleteFact">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete Fact</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView_image"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="button_addPhoto">
          <property name="text">
           <string>Add Photo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_deletePhoto">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete Photo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
